calico_node/Makefile

```diff
@@ -195,7 +198,7 @@ $(NODE_CONTAINER_BIN_DIR)/calico-felix update-felix:
 	-docker rm -f calico-felix
 	# Latest felix binaries are stored in automated builds of calico/felix.
 	# To get them, we create (but don't start) a container from that image.
-	docker pull $(FELIX_CONTAINER_NAME)
+	if $(PULL_FELIX); then docker pull $(FELIX_CONTAINER_NAME); fi
```

Comment:
This is fine - but FWIW I like the shorter:

```
$(PULL_FELIX) || docker pull $(FELIX_CONTAINER_NAME)
```

Reply:
Except that'd be backwards!  It'd pull felix only if the flag was false.

Reply:
D'oh, yes.  Would need to `&&` instead of `||`.

Reply:
But then `false && <something>` has a false return code :-(  I think it needs to be `! $(PULL_FELIX) || ...`, but that's starting to get a bit convoluted.

Reply:
Indeed, it's a lot more fiddly than I thought!  Bearing that in mind, I'm happy that my suggestion doesn't help.